shell/renderer/electron_renderer_client.cc

```diff
@@ -154,6 +154,15 @@ void ElectronRendererClient::DidCreateScriptContext(
   // Load everything.
   node_bindings_->LoadEnvironment(env);
 
+  // Node.js sets a stack trace handler specific to the v8::Context
+  // corresponding to the current Environment. When we're running in a
+  // non-Node.js v8::Context, there will be no correspondent Environment - we
+  // therefore need to unset this handler so that Blink falls back to its
+  // default handling and displays the correct stacktrace.
+  if (prefs.context_isolation) {
+    renderer_context->GetIsolate()->SetPrepareStackTraceCallback(nullptr);
```

Comment:
Don't we need to do the trick we do elsewhere and call into either blinks or nodes handler depending on the current context?  This will just fallback to the default for nodes case as well which we don't want right?

Reply:
I don't think so - we could just fall back in the renderer process unilaterally but the only case that presents issues is when `contextIsolation` is enabled.